metadata.yaml

```diff
@@ -10,6 +10,8 @@ subordinate: false
 provides:
   docker-containers:
     interface: containers
+  events:
```

Comment:
+1 to the new interface, we need to get docs around what its sending added to the readme/docsite.